doc/nrf/protocols/matter/end_product/ecosystems_certification.rst

```diff
@@ -0,0 +1,64 @@
+
+Works with Google Home certification
+************************************
+
+The `Works with Google Home`_ certification program to ensures that a Matter product supports Matter features in the same extent, as the Google Home application.
+A product that passed all the certification test cases is allowed to use the Works with Google Home badge.
+
+The Matter samples delivered in the |NCS| have not received official certificates, as certification can only be obtained for final products.
+However, they have been tested against the Works with Google Home certification test cases and have successfully passed all of them.
+The following is a full list of Matter samples that were verified to pass the Works with Google certification test cases:
+
+* :ref:`matter_lock_sample`
+* :ref:`matter_light_bulb_sample`
+* :ref:`matter_thermostat_sample`
+* :ref:`matter_window_covering_sample`
+* :ref:`matter_weather_station_app`
+
+.. _ug_matter_amazon_certification:
+
+Amazon Frustration-Free Setup certification
+*******************************************
+
+Frustration-Free Setup (FFS) is Amazon's proprietary feature that uses Amazon cloud services to provide the onboarding information necessary to commission the Matter device to a Matter fabric.
+The commissioning process is automatically triggered by the Amazon commissioner and it does not require any user interaction.
+The process starts when the commissioner detects the accessory's Bluetooth LE advertising messages in the proprietary format.
+
+For more information about the FFS and how to enable it in Matter samples, see :ref:`Amazon Frustration-Free Setup configuration<ug_matter_configuring_ffs>` page.
+
+FFS requires passing the additional certification test cases for Matter called `Matter Simple Setup`_.
+There are two types of certification:
+
+* Default - For standard products or for the creation of a reference design.
+* Simplified - For products using a certified reference design.
+
+The Matter platform provided in the |NCS| was tested against the FFS certification test cases and passed them in several variants.
+This means that if you create a Matter product based on the |NCS|, you can approach the simplified certification path.
+You can do this by filling the Reference APID field when you register your product on Amazon's developer page.
+Use one of the following values:
+
++-----------------------------------------------+-----------------------+
+| Matter platform variant                       | Reference Design APID |
++===============================================+=======================+
+| nrf52840dk_nrf52840 (Matter over Thread)      | ZNwt                  |
++-----------------------------------------------+-----------------------+
+| nrf5340dk_nrf5340_cpuapp (Matter over Thread) | xzNd                  |
++-----------------------------------------------+-----------------------+
+| nrf7002dk_nrf5340_cpuapp (Matter over Wi-Fi)  | jyjh                  |
```

Comment:
Should these be board names (in which case the `_cpuapp` part should not be included) or build targets (in which case they should be formatted `nrf52840dk_nrf52840`). For refrence from the the Light bulb sample:
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/056c98ab-32ac-44c9-a8c0-1fa61ee5136e)

Reply:
Fair point, actually I think it should be hardware platforms. I updated the table with this change.